src/Features/TaskTemplates/TaskTemplateOverview/TaskTemplateOverview.tsx

```diff
@@ -354,7 +354,7 @@ export function TaskTemplateOverview({ taskTemplates, updateTemplateInState }) {
 
   if (templateNotFound)
     return (
-      <Error404
+      <EmptyState
         header="Task Template not found"
```

Comment:
Same comment as above - maybe make this consistent by only using title and message?